src/cmd/fetch.go

```diff
@@ -20,29 +21,72 @@ var fetchCmd = &cobra.Command{
 	2. A branch name
 	3. A path to a whitelist file
 	`,
-	
-	Args: cobra.MinimumNArgs(3),
+
+	//Args: cobra.MinimumNArgs(3),
 
 	Run: func(cmd *cobra.Command, args []string) {
-		if len(args) % 3 != 0 {
+		branch, err := rootCmd.Flags().GetString("branch")
+		if err != nil {
+			panic(err)
+		}
+		whitelist, err := rootCmd.Flags().GetString("whitelist")
+		if err != nil {
+			panic(err)
+		}
+
+		if len(args)%3 != 0 && branch == "" && whitelist == "" {
 			panic("arguments is not divisable by 3")
```

Comment:
Not a big fan of this error message.
This seams like the more standard way to do this kind of error message:
```suggestion
			panic("Wrong number of arguments")
```

Reply:
I would prefer this

```suggestion
			panic("Wrong number of arguments expected 3: <Url> <Branch> <Path/to/whitelist_file>")
```